build_kernel_w_cma.sh

```diff
@@ -19,6 +19,8 @@ if [ $ret -eq 0 ]; then
     exit 0
 fi
 
+sed -e
```

Comment:
I think this should have been
```suggestion
set -e
```